### PR TITLE
DOCS: Add docs for bigquery gem for read and write

### PR DESCRIPTION
**Pull request by kishoreypro**

### CODE REVIEW

File: docs/low-code-spark/gems/source-target/warehouse/bigquery.md [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+  - gems
+  - warehouse
+  - bigquery
+---
+
+:::info Built on
+This connector is built on top of the already available [`spark-bigquery-connector`](https://github.com/GoogleCloudDataproc/spark-bigquery-connector/)
+
+To use this Gem in Prophecy, `com.google.cloud.spark:spark-3.3-bigquery-pushdown_2.12:0.30.0` Maven external dependency needs to be installed on cluster if it's not present.
+For installing dependencies from Prophecy UI, Please check [dependency management docs](https://docs.prophecy.io/low-code-spark/extensibility/dependencies)
```
Comment (jainabhinav) [inline docs/low-code-spark/gems/source-target/warehouse/bigquery.md:16]: Rephrase it to for non-databricks clusters we need to install the corresponding library and add the link to the library compatiblity matrix in the docs

**Review by jainabhinav — APPROVED** [review]

**Review by rishabh-sudan — APPROVED** [review]
